### PR TITLE
Feature/add start end holds

### PR DESCRIPTION
**Pull request by yarkhinephyo**
#### Summary
- Add new toggle when placing handhold
  -  `0-HandStart` vs `1-HandStart` vs `2-HandStart`

#### Minor
- Reduce numbering fontsize
- Increase minimal size of drawn boxes to 3 x 3 pixels
- Change name from `Hide Number` to `Hide Order`
- Change naming from `enumSelectModes` to `enumSelectMode`

#### New Layout

image

### CODE REVIEW

File: user_interface/src/common/konva/drawLayer.js [hunk 1]
```diff
@@ -107,11 +107,11 @@ export const addKonvaDrawLayer = (stageNode) => {
       listening: false,
     });
 
-    // Do not add if the DrawRect is too small (Less than 1 pixel)
-    if (drawRect.width() > 1 && drawRect.height() > 1) {
+    // Do not add if the DrawRect is too small (Less than 3 pixel)
```
Comment (qwoprocks) [inline user_interface/src/common/konva/drawLayer.js:110]:
```suggestion
    // Do not add if the DrawRect is too small (Less than 3 pixels)
```

Comment (qwoprocks) [reply user_interface/src/common/konva/drawLayer.js:110]: I think this check should be moved up, before creating newRect. If not there'll be wasted work

File: user_interface/src/components/ModeSelector.vue [hunk 2]
```diff
@@ -20,8 +20,11 @@
           <b-button v-if="!isSelectModeDrawBox" @click="onReset" variant="outline-info"
             >Reset</b-button
           >
-          <b-button v-if="!isSelectModeDrawBox" @click="toggleShowNumbers" variant="outline-info"
-            >{{ this.getShowNumberMode ? 'Hide' : 'Unhide' }} Numbers</b-button
+          <b-button v-if="!isSelectModeDrawBox" @click="toggleShowOrder" variant="outline-info"
+            >{{ this.showOrderMode ? 'Hide' : 'Unhide' }} Order</b-button
+          >
+          <b-button v-if="!isSelectModeDrawBox" @click="toggleHandStart" variant="outline-info"
+            >{{ this.handStartMode }}-HandStart</b-button
```
Comment (qwoprocks) [inline user_interface/src/components/ModeSelector.vue:27]: This should actually be HoldStart instead of hand start.

**Review by qwoprocks — CHANGES_REQUESTED** [review]

File: user_interface/src/components/ModeSelector.vue (hunk 2 above)
Comment (yarkhinephyo) [reply user_interface/src/components/ModeSelector.vue:27]: Now only got handstart. Holdstart may not may not be confusing LOL (Imagine all the markings). But yeah I get that its more similar to comp style? Can do nxt PR

File: user_interface/src/common/enumHandStartMode.js [hunk 3]
```diff
@@ -0,0 +1,23 @@
+/**
+ * ENUM
+ * https://stackoverflow.com/questions/44447847/enums-in-javascript-with-es6
+ */
+class HandStartMode {
+  // Private Fields
+  static #_NOSHOW = 0;
```
Comment (qwoprocks) [inline user_interface/src/common/enumHandStartMode.js:7]: Maybe can remove the underscore?

**Review by qwoprocks — APPROVED** [review]

**Pull request merged**